### PR TITLE
updated proto

### PR DESCRIPTION
**Pull request by andreiSaw**

### CODE REVIEW

**Comment by andreiSaw** [comment]
added last_modified job field @sashabaranov

**Comment by andreiSaw** [comment]
also batch import fixed

File: disneyland.proto [hunk 1]
```diff
@@ -29,25 +29,19 @@ message RequestWithId {
...
     rpc GetJob (RequestWithId) returns (Job) {}
     rpc ListJobs (ListJobsRequest) returns (ListOfJobs) {}
-
+    rpc CreateMultipeJobs (ListOfJobs) returns (ListOfJobs) {}
     rpc ModifyJob (Job) returns (Job) {} /** Merges passed Status, MetricValue and Metadata to db */
-
     rpc PullPendingJobs (ListJobsRequest) returns (ListOfJobs) {}
     rpc DeleteJob(RequestWithId) returns (Job) {}
-    //rpc BidiJobs (stream ListOfJobs) returns (stream ListJobStatusResponse) { }
+    rpc BidiJobs(stream ListJobsRequest) returns(stream Job) {}
```
Comment (sashabaranov) [inline disneyland.proto]: Bidirectional rpc should be expanded.

Server should be able to receive

- Information about worker — cpu/memory available/how many computational slots available
- Information about running jobs — cpu/memory usage, logs(if requested)

Client should be able to receive:

- New jobs
- Job cancelation signal

Also both client and server should be able to ping/pong each other.

Having all this information exchanges enables server to execute some smart scheduling decisions.

**Comment by andreiSaw** [comment]
In a bidirectional streaming RPC, again the call is initiated by the client calling the method and the server receiving the client metadata, method name, and deadline. Again the server can choose to send back its initial metadata or wait for the client to start sending requests.

What happens next depends on the application, as the client and server can read and write in any order - the streams operate completely independently. So, for example, the server could wait until it has received all the client’s messages before writing its responses, or the server and client could “ping-pong”: the server gets a request, then sends back a response, then the client sends another request based on the response, and so on.
https://grpc.io/docs/guides/concepts.html

**Comment by andreiSaw** [comment]
#3

**Comment by andreiSaw** [comment]
Bidirectional rpc should be expanded.

Server should be able to receive

Information about worker — cpu/memory available/how many computational slots available
Information about running jobs — cpu/memory usage, logs(if requested)
Client should be able to receive:

New jobs
Job cancelation signal
Also both client and server should be able to ping/pong each other.

Having all this information exchanges enables server to execute some smart scheduling decisions.

**Pull request merged**